src/Web/Spock/Internal/Cookies.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE CPP               #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards   #-}
+
+module Web.Spock.Internal.Cookies
+    ( CookieSettings(..)
+    , defaultCookieSettings
+    , CookieEOL(..)
+    , generateCookieHeaderString
+    )
+where
+
+import           Control.Monad.IO.Class (liftIO)
+import           Data.Monoid            ((<>))
+import qualified Data.Text              as T
+import           Data.Time
+#if MIN_VERSION_time(1,5,0)
+#else
+import           System.Locale          (defaultTimeLocale)
+#endif
+
+data CookieSettings = CookieSettings { csEOL      :: CookieEOL
```

Comment:
I use the underscore record label style in Spock, would be great if you could follow that 'convention' too to keep the code uniform. (I should add a coding conventions file sometime...)